protocol_connect.go

```diff
@@ -47,6 +47,10 @@ const (
 	connectUnaryContentTypePrefix     = "application/"
 	connectUnaryContentTypeJSON       = connectUnaryContentTypePrefix + "json"
 	connectStreamingContentTypePrefix = "application/connect+"
+
+	connectUnaryEncodingQueryParameter = "enc"
+	connectUnaryMessageQueryParameter  = "msg"
+	connectUnaryBase64QueryParameter   = "b64"
```

Comment:
Didn't we also decide that we need the connect protocol version as a query parameter too (for middle boxes)?

Reply:
Fixed in d84714f940ed601f60967ff94166d393b9883909.